gatsby-config.ts

```diff
@@ -11,6 +11,7 @@ const siteMetadata: SiteMetadata = {
   company: `GreenOn Software`,
   siteUrl: `https://4markdown.com`,
   description: `Craft, edit, and preview your Markdown documents with ease, all within a clean and user-friendly interface`,
+  keywords: `Markdown, 4markdown, Editor, Github, GreenOnSoftware, Open Source, Node.js, Markdown preview, online editor,`,
```

Comment:
add here: "Online markdown editor", change to 4Markdown, add "Programming articles"

Reply:
Changed.